web/ui/react-app/src/pages/Targets.tsx

```diff
@@ -1,7 +0,0 @@
-import React, { FC } from 'react';
```

Comment:
If we're moving components used by the targets page into a subdirectory, we'll also want to move other components in the main `/src` directory into their respective page / utils / etc. directories, but let's not do the latter in this PR yet.

Reply:
Cool. I think it would be nice to have page-specific components in the `src/pages/<page>/` directory and shared components in `src/components`